tests/test_benchmarking.py

```diff
@@ -1,44 +1,33 @@
 """Tests for config files in benchmarking/ folder."""
+import glob
 import os
-import subprocess
+import pathlib
 
 import pytest
 
-from imitation.data import types
+from imitation.scripts import train_adversarial, train_imitation
 
-BENCHMARKING_DIR = types.parse_path("benchmarking")
-
-if not BENCHMARKING_DIR.exists():  # pragma: no cover
-    raise RuntimeError(
-        "The benchmarking/ folder has not been found. Make sure you are "
-        "running tests relative to the base imitation project folder.",
-    )
+THIS_DIR = pathlib.Path(__file__).absolute().parent
+BENCHMARKING_DIR = THIS_DIR / ".." / "benchmarking"
```

Comment:
```suggestion
BENCHMARKING_DIR = THIS_DIR.parent() / "benchmarking"
```

Reply:
parent not parent() but yeah that's nicer.